src/ui/widgets/Image/image.tsx

```diff
@@ -54,8 +54,8 @@ export const ImageComponent = (
     ...commonCss(props as any),
     overflow,
     textAlign: "left",
-    width: imageWidth,
-    height: imageHeight
+    width: "100%",
```

Comment:
If we're setting the width and height here, we can probably remove the if-else statement on lines 44-51 that sets it based on `stretchToFit` and `fitToWidth`/`fitToHeight`, and the unused `imageHeight`/`imageWidth` variables.